core/src/main/java/org/bitcoinj/coinjoin/CoinJoinBroadcastTx.java

```diff
@@ -172,4 +186,16 @@ public boolean isValidStructure() {
         }
         return allOf;
     }
+    @VisibleForTesting
+    public boolean sign(BLSSecretKey blsKeyOperator) {
+        Sha256Hash hash = getSignatureHash();
+
+        BLSSignature sig = blsKeyOperator.Sign(hash);
+        if (!sig.isValid()) {
+            return false;
+        }
+        signature = new MasternodeSignature(sig.bitcoinSerialize());
+
+        return true;
+    }
```

Comment:
The client would never sign such messages, but for our `CoinJoinSessionTest` the `CoinJoinServer` needs to sign some of the messages that are sent to the clients.